### ISSUE TITLE
Add support for modularscale-sass 3.x

### ISSUE THREAD

**Issue by patrickcate** [zellwk/typi#44]
It would be nice if Typi supported the latest 3.x version of [modularscale-sass](https://github.com/modularscale/modularscale-sass).

**Comment by zellwk** [comment]
Sounds good. Let me find time for this. You could also send a PR if you're cool with that :)

**Comment by strarsis** [comment]
@zellwk: Is the branch 'three' in this repository the WIP branch for modular scale 3.x integration?

**Comment by zellwk** [comment]
@strarsis nope. I don't think so. It's been so long since I worked on Typi that I can't really remember, unfortunately.

**Comment by dcdeiv** [comment]
I'm using Bourbon's modular-scale and it works flawlessly

**Comment by strarsis** [comment]
@zellwk: Are there plans for finishing support for `modular scale` 3.x in the near future? 🐱

**Comment by zellwk** [comment]
Nope. I don't have the space to work on this project for the upcoming half year 😢
